content/influxdb/cloud-dedicated/admin/query-system-data.md

```diff
@@ -119,6 +128,112 @@ Replace the following:
 When prompted, enter `y` to acknowledge the potential impact querying system
 tables may have on your cluster.
 
+### Optimize queries to reduce impact to your cluster
+
+Querying InfluxDB v3 system tables may impact the performance of your
+{{< product-name omit=" Clustered" >}} cluster.
+As you write data to a cluster, the number of partitions and Parquet files
+can increase to a point that impacts system table performance.
+Queries that took milliseconds with fewer files and partitions might take 10
+seconds or longer as files and partitions increase.
+
+Use the following filters to optimize your system table queries and reduce the impact on your
+cluster's performance.
+
+##### Filter by table name
+
+Filter by `table_name` when querying `system.tables`, `system.partitions`, or `system.compactor`. 
+
+```sql
+WHERE table_name = 'TABLE_NAME' (string)
+```
+
+##### Filter by partition key 
+
+Filter by `partition_key` when querying `system.partitions` or `system.compactor`.
+
+```sql
+WHERE partition_key = 'PARTITION_KEY (string)
+```
+
+To further improve performance, use `AND` to pair `partition_key` with `table_name`--for example: 
+   
+```sql
+SELECT * 
+FROM system.partitions 
+WHERE table_name = 'TABLE_NAME' 
+ AND partition_key = 'PARTITION_KEY';
+```
+
+- `PARTITION_KEY`: a [partition key]((/influxdb/cloud-dedicated/admin/custom-partitions/#partition-keys))
+   derived from the table's partition template.
+
+###### Partition key format
+
+The default format for `partition_key` is `%Y-%m-%d` (for example, `2024-01-01`).
+If the table uses a custom partition template, the `partition_key` format will differ.
+
+##### Filter by partition ID 
+
+Filter by `partition_id` when querying `system.partitions` or `system.compactor`.
+
+```sql
+WHERE partition_id = PARTITION_ID (int64)
+```
```

Comment:
~~~suggestion
When querying the `system.partitions` or `system.compactor` table, use the `WHERE` clause to
filter by `partition_id` .

```sql
SELECT * FROM system.partitions WHERE partition_id = PARTITION_ID
```
~~~